acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -32,20 +30,10 @@
 
 install_packages_on(hosts, PACKAGES)
 
+# make sure install is sane, beaker has already added puppet and ruby
+# to PATH in ~/.ssh/environment
 agents.each do |agent|
-  case agent['platform']
-  when /windows/
-    arch = agent[:ruby_arch] || 'x86'
-    base_url = ENV['MSI_BASE_URL'] || "http://builds.puppetlabs.lan/puppet-agent/#{ENV['SHA']}/artifacts/windows"
```

Comment:
@Iristyle yeah good point, I punted on that issue because I was thinking we never use that capability in CI. But I forgot about the one-off developer build issue.